src/main/scala/com/normation/cfclerk/xmlparsers/SectionSpecParser.scala

```diff
@@ -101,6 +101,19 @@ class SectionSpecParser(variableParser:VariableSpecParser) extends Loggable {
         case _ => //OK
       }
 
+
+      /*
+       * Check that all "used" variable are (at least) defined elsewhere in the technique
```

Comment:
We, here in the internal, we have usedFields, so it makes sens to keep "used". Derived is a dedicated term used only by masterPassword, with a limited semantic on that case. As the code here is not specific to password, I would like to keep something not bounded to it.